app/assets/stylesheets/common/base/compose.scss

```diff
@@ -307,6 +307,11 @@
         margin-right: 5px;
       }
     }
+    #draft-status .d-icon-user-edit {
+      color: $danger;
+      font-size: 20px;
```

Comment:
we should use our $font-size macro

Reply:
Not really making any sense here since this is only affecting the size of the icon and no text.

Reply:
Yes I kind of agree, but it helps to keep every thing in sync if people change some value.

What it basically means is: this should be bigger than this.

Reply:
Sure, yeah. But I don't really get the relation of making text smaller by design and then also making this icon smaller so that no one can see it. I think it should have a fixed size.

Just wanted to say my opinion. If you still want it changed, I can of course do that.

Reply:
I don’t have strong feelings on this, we already do this at various places (and we also have fixed sizes have various places). I do think it's better, but let it as it is, fine :p